docker/manylinux-builder/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM quay.io/pypa/manylinux2014_x86_64
+ARG base_image
+FROM $base_image
```

Comment:
What is docker convention for arg naming, I thought that BASE_IMAGE would be more in line what I have seen before.

Reply:
> What is docker convention for arg naming

Lowercase build args are preferred because this differentiates them from the environment variables, which are always in uppercase.

Reply:
Ok